qlobber-sub.js

```diff
@@ -23,14 +23,26 @@ QlobberSub.prototype._add_value = function (clientMap, val) {
   topicMap.set(val.topic, val.qos)
 }
 
-QlobberSub.prototype._add_values = function (dest, clientMap) {
+QlobberSub.prototype._add_values = function (dest, clientMap, topic) {
   for (var clientIdAndTopicMap of clientMap) {
-    for (var topicAndQos of clientIdAndTopicMap[1]) {
-      dest.push({
-        clientId: clientIdAndTopicMap[0],
-        topic: topicAndQos[0],
-        qos: topicAndQos[1]
-      })
+    var clientId = clientIdAndTopicMap[0]
+    var topicMap = clientIdAndTopicMap[1]
+    if (topic === undefined) {
+      for (var topicAndQos of topicMap) {
+        dest.push({
+          clientId: clientId,
+          topic: topicAndQos[0],
+          qos: topicAndQos[1]
+        })
+      }
```

Comment:
Btw @mcollina do you know if there's any regularly-updated list of what's performant in Node and what's not?

I'm in a state of constant confusion as to what to use. Also, what if `var of` become faster in future versions? And why is it so slow?